argo_rollouts/using-cluster-scoped-rollouts-instance-to-manage-rollouts-resources.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="using-cluster-scoped-rollouts-instance-to-manage-rollouts-resources"]
+= Using a cluster-scoped rollouts instance to manage rollouts resources
+context: using-cluster-scoped-rollouts-instance-to-manage-rollouts-resources
+
+toc::[]
+
+By default, Argo rollouts supports the cluster-scoped mode of installation for Argo rollouts custom resources (CRs). This mode of installation uses the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable to specify a list of namespaces that can be used to manage the rollout resources. 
```

Comment:
Should we also talks how to disable cluster-scoped and install namespace-scoped instance?

Reply:
Hey @jparsai - We have already documented the installation of namespace-scoped instance before this section. There is a lot of content that talks about how to install it. Would that suffice? PTAL.